qcfractal/alembic/versions/1604623c481a_id_pirmary_key_for_torsion_init_mol.py

```diff
@@ -0,0 +1,35 @@
+"""id(pirmary key) for torsion_init_mol
+
+Revision ID: 1604623c481a
+Revises: fb5bd88ae2f3
+Create Date: 2020-07-02 18:42:17.267792
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "1604623c481a"
+down_revision = "fb5bd88ae2f3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    op.execute(
+        "DELETE FROM torsion_init_mol_association a USING \
+    (SELECT MIN(ctid) as ctid, torsion_id, molecule_id \
+        FROM torsion_init_mol_association \
+        GROUP BY torsion_id, molecule_id HAVING COUNT(*) > 1 \
+      ) b \
+      WHERE a.torsion_id = b.torsion_id and a.molecule_id = b.molecule_id \
+      AND a.ctid <> b.ctid"
+    )
+
+    op.execute("alter table add primary key (torsion_id, molecule_id)")
```

Comment:
Are you missing the table name here? I get a syntax error when running

(also possibly missing from the downgrade)

Reply:
I just saw this...yeah it's missing, I'll pull and reformat the code.

Reply:
This has already been fixed and merged. See the commit below

https://github.com/MolSSI/QCFractal/commit/842b128dd4fb3b93492578008de0969e85e3039a